src/Badger/GameBundle/Entity/Step.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Badger\GameBundle\Entity;
+
+/**
+ * Step
+ *
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class Step implements StepInterface
```

Comment:
Maybe we should name this entity `AdventureStep` as it's really related to Adventures. Otherwise I think it's a bit too generic, what do you think?